hyperon_das_atomdb/adapters/redis_mongo_db.py

```diff
@@ -85,6 +89,35 @@ def __str__(self):
         return str(self.base)
 
 
+class MongoDBIndex(Index):
+    def __init__(self, collection: Collection) -> None:
+        self.collection = collection
+
+    def create(self, field: str, **kwargs) -> Tuple[str, Any]:
+        conditionals = None
+
+        for key, value in kwargs.items():
+            key = 'named_type' if key == 'type' else key
```

Comment:
This is not correct. Whoever called this method should have changed `type` by `named_type` already. You can't make this change here as it breaks the orthogonality of the key passing.

Reply:
Done